docs/developers_guide/organization.md

```diff
@@ -2022,3 +2022,34 @@ A developer  defines a test suite by creating a `.txt` file within the
 `polaris/<component>/suites` directory.  The format of the `.txt` file is a 
 list of the work directories to  the tests desired to be part of the suite.  A 
 line starting with `#` will be  treated as a comment line.
+
+The philosophy and requirements for the test suites are as follows:
+
+### Pull-Request (PR) suite
+
+The PR suite is intended to be run in the context of its namesake, a pull-
+request to the model component. It should require no more than 32 cores (the
+minimum number of cores on a typical compute node) and take no more than 20
+minutes of run time on 32 cores.
+
+Examples of test cases that may be included in this suite:
+
+* Short tests to compare the solution and timers with a baseline (often called
+performance tests lasting 3-5 timesteps)
+* Variants on the above with different combinations of config options
+* Tests of all common config option combinations
```

Comment:
I don't know if we want to mention tests that verify identical behavior:
* across different numbers of cores
* across different numbers of threads
* with a longer model run and 2 (or more) shorter runs of the same total duration with a model restart

Reply:
@xylar Thanks for the suggestion. Yes, I agree and added those tests.